### DISCUSSION TITLE
Why were the patch versions for CVE-2020-15106, CVE-2020-15112 and CVE-2020-15113 released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [etcd-io/etcd#16945] [Q&A]
We are a research team dedicated to Golang, have discovered that CVE-2020-15106, CVE-2020-15112 and CVE-2020-15113 were addressed in commit c716d764d0edd47e898fc58607afa91164687574, 0d5c692eb6e824beb135693605458da832ca44cb and e06006ea28a2b25319d4bc0d94d94079971a39af. However, upon analyzing the commit, we observed that the patch version (v3.3.23) was released after a lapse of over one month. We are interested in understanding the reasons behind this delay in releasing the patch version, as it could potentially impede the prompt dissemination of patches to downstream users. We seek clarification on whether the delay might be attributed to:

1. Issues with testing and CI checking.
2. Other commits requiring inclusion in a single release.
3. By convention, infrequent release of versions.
4. Other reasons.
   We appreciate your attention to this matter and eagerly await your response. Thank you.

**Comment by serathius** [comment]
> We are interested in understanding the reasons behind this delay in releasing the patch version, as it could potentially impede the prompt dissemination of patches to downstream users

Agree, users should take better care of open source projects they use. Contributions are welcomed!
